packages/daf-core/src/graphql/graphql-core.ts

```diff
@@ -21,6 +24,7 @@ import { Identity } from '../entities/identity'
 
 export interface Context {
   agent: Agent
+  authenticatedDid?: Promise<string>
```

Comment:
Why is this `Promise<string>` and not just a simple `string`?

I was thinking, that it can be used like that:

``` typescript
const server = new ApolloServer({
  typeDefs: [Daf.Gql.baseTypeDefs, Daf.Gql.Core.typeDefs],
  resolvers: merge(Daf.Gql.Core.resolvers),
  context: async ({ req }) => {
    const authenticatedDid = await applicationSpecificLogic(req.headers.authorization)
    return { agent, authenticatedDid }
  }
})
```

Reply:
the context function can't be async.  Because verifying a jwt is an async function, this has to be a promise.

Reply:
https://github.com/apollographql/apollo-server/issues/225

Reply:
Your mentioned issue is almost 4 years old. I just tried it and it works fine:
https://github.com/uport-project/daf/blob/master/examples/id-hub/src/index.ts#L50

Reply:
It was causing linting warning in my ide when I tried before, now its not, must have been something wrong with my config. My mistake, will change.